s3/settings.py

```diff
@@ -0,0 +1,29 @@
+"""
+Configuration for the S3 Interface microservice settings.
+"""
+import json
+import logging
+import os
+
+log = logging.getLogger(__name__)
+
+def get_s3_info():
+    vcap_services = os.getenv("VCAP_SERVICES", "")
```

Comment:
Since this method is pretty dependent on there actually being something in `VCAP_SERVICES`, we probably shouldn't just keep going if it was empty. Perhaps the following?
```suggestion
    vcap_services = os.getenv("VCAP_SERVICES")
    if vcap_services is None:
        # The variable didn't exist so we should at least log that and bail if not provide a useful error of some sort.
```